src/main/java/org/minions/devfund/chelo/Movie.java

```diff
@@ -0,0 +1,77 @@
+package org.minions.devfund.chelo;
+
+import java.util.ArrayList;
+
+/**
+ * Movie class.
+ */
+public class Movie {
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    /**
+     * Movie constructor.
+     * @param name name.
+     */
+    public Movie(final String name) {
+        this.name = name;
+        this.rating = 0.0;
+        actors = new ArrayList<>();
+    }
+
+    /**
+     * Movie constructor.
+     */
+    public Movie() {
+        this("");
+    }
+
+    /**
+     * Getter for movie name.
+     * @return name.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Setter for Movie name.
+     * @param name name.
+     */
+    public void setName(final String name) {
+        this.name = name;
+    }
+
+    /**
+     * Getter for actor list.
+     * @return actors.
+     */
+    public ArrayList<Actor> getActors() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The return type of this method should be an interface such as "List" rather than the implementation "ArrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)